human_biomolecular_data/data_sources.md

```diff
@@ -1,4 +1,55 @@
 ---
-title: Data sources
-description: Finding and sharing data
+title: Human biomolecular data sources
```

Comment:
I would not change this title since it will be duplicated Human biomolecular data section title above it :)

Reply:
okay! then is it okay that all pages are called the same?

Reply:
The current situation is that the pages have the same name, but they have different URLs and different page_id